src/main/java/org/liquidengine/legui/binding/accessor/selectbox/SelectBoxElementsAccessor.java

```diff
@@ -0,0 +1,51 @@
+package org.liquidengine.legui.binding.accessor.selectbox;
+
+import com.google.gson.reflect.TypeToken;
+import java.lang.reflect.Type;
+import java.util.List;
+import org.liquidengine.legui.binding.accessor.AbstractFieldAccessor;
+import org.liquidengine.legui.component.SelectBox;
+
+/**
+ * Accessor for selectbox elements.
+ *
+ * @author Aliaksandr_Shcherbin.
+ */
+public class SelectBoxElementsAccessor extends AbstractFieldAccessor<SelectBox, List<String>> {
```

Comment:
You mean create constructor which allow to initialize FieldAccessor with lambdas?
It could be created for direct use in Java but we still need implementations for use in binding descriptor (XML).